packages/perseus/src/widgets/expression/expression.test.tsx

```diff
@@ -501,54 +501,6 @@ describe("Expression Widget", function () {
         });
     });
 
-    describe("error tooltip", () => {
-        beforeEach(() => {
-            jest.spyOn(Dependencies, "getDependencies").mockReturnValue(
-                testDependencies,
-            );
-            jest.useFakeTimers();
-        });
-
-        it("shows error text in tooltip", async () => {
-            // Arrange
-            const {renderer} = renderQuestion(expressionItem2.question);
-            const expression = renderer.findWidgets("expression 1")[0];
-
-            // Act
-            // Note(jeremy): You might think you could collapse all of these calls
-            // inside a single act() block, but that didn't work. The only way this
-            // test passes is with each statement in its own act() call. :/
-            act(() => expression.insert("x&&&&&^1"));
-            act(() => jest.runOnlyPendingTimers());
-            act(() => screen.getByRole("textbox").blur());
-            act(() => jest.runOnlyPendingTimers());
-
-            // Assert
-            await waitFor(() =>
-                expect(
-                    screen.getByText("Oops! Sorry, I don't understand that!"),
-                ).toBeVisible(),
-            );
-        });
-
-        it("does not show error text when the sen() function is used (Portuguese for sin())", async () => {
-            // Arrange
-            const {renderer} = renderQuestion(expressionItem2.question);
-            const expression = renderer.findWidgets("expression 1")[0];
-
-            // Act
-            act(() => expression.insert("sen(x)"));
```

Comment:
Wonder if it's worth adding "sen(x)" to the `scoreExpression` tests or something so we're not losing a check.

Reply:
good eye ty